examples/dataset/sdohmi_dataset.ipynb

```diff
@@ -0,0 +1,177 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
```

Comment:
Not sure if we actually want this anymore. The combined AIA/HMI notebooks is sufficient and structures the examples a bit better:

- AIA (Sunpy maps)
   - AIA+HMI (Sunpy maps)
   - AIA+EVE (Time series)